### PR TITLE
[feat] upgrade to vue 3

### LINKED ISSUE #3: Upgrade to Vue 3
Move the codebase to use Vite and Vue 3 instead of Vue CLI(Webpack) and Vue 2

### PR DESCRIPTION
**Pull request by DominusKelvin**
This PR closes #3 

It also updates the PayPal SDK to the latest version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,38 +3,37 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "start": "npm run serve",
-    "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "dev": "vite",
+    "build": "vite build",
+    "server": "vite preview",
```
Comment (gregjopa) [inline package.json:8]: It's awesome to see vite being used!

**Review by gregjopa — APPROVED** [review]
Thanks for helping improve this vue example repo @DominusKelvin! Great work 💯 

If you're up for it, I have an idea for a future PR contribution. It's to add server code to this example similar to this react sample code: https://github.com/paypaldev/PayPal-React-FullStack-Standard-Checkout-Sample. Our new integration pattern is to recommend developers to call the Orders API from their server-side code so the order dollar amount stays on the server side. This React example also uses vite. I think you can copy and paste the server-side code from this repo along with most of the vite config and apply it here.

File: package.json (hunk 1 above)
Comment (DominusKelvin) [reply package.json:8]: Yeah it is awesome!

**Comment by DominusKelvin** [comment]
> Thanks for helping improve this vue example repo @DominusKelvin! Great work 💯
> 
> If you're up for it, I have an idea for a future PR contribution. It's to add server code to this example similar to this react sample code: https://github.com/paypaldev/PayPal-React-FullStack-Standard-Checkout-Sample. Our new integration pattern is to recommend developers to call the Orders API from their server-side code so the order dollar amount stays on the server side. This React example also uses vite. I think you can copy and paste the server-side code from this repo along with most of the vite config and apply it here.

Yeah that sounds interesting. I'll take a look

**Comment by awaisalwaisy** [comment]
May be you forgot this line to fix.

**Pull request merged**